CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.11.0
+
+- Adds support for Conjur v5.
```

Comment:
Can you add something like this to the changelog?

`CONJUR_VERSION`: Conjur version (4 or 5, defaults to 5)

that way we tell users in changelog what they're going to have to update, or about new variables.

Reply:
We should probably also mention here and elsewhere that they need to use `'4'` and not `4`. Using the int breaks parsers right?